### PR TITLE
app-admin/syslog-summary: add sys-apps file suggestion

### PR DESCRIPTION
**Pull request by lmiphay**
This adds a suggestion to install sys-apps/file[python] to support processing gzip compressed logfiles.

Also:
EAPI 6
Dropped stable keywords

I would like to maintain this package, and have added myself to metadata.xml

Closes: https://bugs.gentoo.org/472884

### CODE REVIEW

File: app-admin/syslog-summary/syslog-summary-1.14-r2.ebuild [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND=""
+RDEPEND="${PYTHON_DEPS}"
+
+src_prepare() {
+	python_fix_shebang -f syslog-summary
+
+	sed -i -e 's:python-magic:sys-apps/file[python]:' "syslog-summary"
```
Comment (monsieurp) [inline app-admin/syslog-summary/syslog-summary-1.14-r2.ebuild]: There's a missing `|| die` here. Please add it.

File: app-admin/syslog-summary/syslog-summary-1.14-r2.ebuild [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit eutils python-single-r1
```
Comment (monsieurp) [inline app-admin/syslog-summary/syslog-summary-1.14-r2.ebuild]: I believe `eutils` isn't required in this ebuild. You can nix it.

File: app-admin/syslog-summary/syslog-summary-1.14-r2.ebuild [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+	# Sadly, the makefile is useless for us.
+	rm Makefile || die
+
+	eapply_user
+}
+
+src_install() {
+	dobin syslog-summary
+	dodoc AUTHORS ChangeLog NEWS README
```
Comment (monsieurp) [inline app-admin/syslog-summary/syslog-summary-1.14-r2.ebuild]: Remove this line and call `einstalldocs` instead.

**Comment by lmiphay** [comment]
Thanks for the review @monsieurp

File: app-admin/syslog-summary/syslog-summary-1.14-r2.ebuild [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit python-single-r1
+
+DESCRIPTION="Summarizes the contents of a syslog log file"
+HOMEPAGE="https://github.com/dpaleino/syslog-summary"
+SRC_URI="mirror://github/dpaleino/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-3"
```
Comment (mgorny) [inline app-admin/syslog-summary/syslog-summary-1.14-r2.ebuild]: Could you please verify the license? Look at the copyright notice on top of source files. If it happens to state '[GPL] version 3 or (at your blah blah) newer', then it should be `GPL-3+` instead.

Comment (lmiphay) [reply app-admin/syslog-summary/syslog-summary-1.14-r2.ebuild]: thanks for spotting that @mgorny - now fixed.

**Comment by lmiphay** [comment]
For reference there is distutils python 3.6 (and python 2.7) ebuild here: 

https://cgit.gentoo.org/user/lmiphay.git/tree/app-admin/syslog-summary/syslog-summary-1.14-r4.ebuild

**Pull request closed without merging**